server/src/CoqProject.ts

```diff
@@ -86,9 +86,12 @@ export class CoqProject {
       this.coqProjectRoot = newSettings.coq.coqProjectRoot;
       this.console.log("Updated project root to " + this.getCoqProjectRoot());
     }
-    if(newSettings.coq.loadCoqProject ) {
+    if(newSettings.coq.loadCoqProject && !newSettings.coqtop.useDune) {
```

Comment:
does this means that if your flag is not set, everything works like as before?

Reply:
Yes, that was my design goal :-).